source/Server.Contracts/ServiceMessages/ICreateTargetServiceMessageHandler.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using Octostache;
+using Sashimi.Server.Contracts.Accounts;
+using Sashimi.Server.Contracts.Endpoints;
+
+namespace Sashimi.Server.Contracts.ServiceMessages
+{
+    public interface ICreateTargetServiceMessageHandler : IServiceMessageHandler
+    {
+        Endpoint BuildEndpoint(IDictionary<string, string> messageProperties, VariableDictionary variables, Func<string, string> accountIdResolver, Func<string, string> certificateIdResolver, Func<string, string> workerPoolIdResolver, Func<string, AccountType> accountTypeResolver);
```

Comment:
Is this enough arguments to warrant a class to resolve these? thoughts @johnsimons